app/controllers/concerns/foreman/controller/provisioning_templates.rb

```diff
@@ -59,6 +59,7 @@ def global_registration_vars
       organization: organization,
       location: location,
       hostgroup: host_group,
+      insecure: params['insecure'] == 'true',
```

Comment:
Shouldn't we use something like `ActiveRecord::Type::Boolean.new.deserialize` to also accept other values which are usually considered truthy, such as `TRUE`, `1` and so on?